site/public/js/forum.js

```diff
@@ -1679,6 +1679,7 @@ function loadThreadHandler(){
         event.preventDefault();
         var obj = this;
         var thread_id = $(obj).data("thread_id");
+        var thread_title = $(obj).data("thread_title");
```

Comment:
It's good practice to use `let` and `const` in JS files.  The rest of this file hasn't been linted since `eslint` was added which is why the other variables are defined with the `var` keyword.
```suggestion
        const thread_title = $(obj).data("thread_title");
```